src/net/dash/segment_loader.ts

```diff
@@ -112,9 +109,10 @@ const segmentPreLoader = (customSegmentLoader? : CustomSegmentLoader) => ({
     return EMPTY;
   }
 
-  // construct url for the segment
-  const path = media ? replaceTokens(media, segment, representation) : "";
-  const url = resolveURL(representation.baseURL, path);
+  if (!media) {
+    log.warn("Couldn't load segment" + segment.id + " because no URL is defined.");
```

Comment:
I'm not sure we should actually return ``EMPTY`` in those kind of conditions.

As it is define in typings as legal, no media should more mean ``nothing to download`` that ``nothing to do``, no?

In that case, maybe we should just construct a ``null`` segment. What do you think?

Returning empty will not trigger the parser next (and may break things).

I have no idea of which type of situation could lead to that though.

What do you think?